HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -77,14 +77,14 @@ x[y]; // returns [10, 10, 30, 30, 20]
 ::
 
 method::clipAt
-Same as link::#-at::, but values for strong::index:: greater than the size of the ArrayedCollection will be clipped to the last index.
+Same as link::#-at::, but items for strong::index:: greater than the link::Classes/ArrayedCollection:: instance size minus one will be clipped to code::size - 1::, which is the last index.
```

Comment:
I'm pretty sure it should still save `values for index` and not `items for index`. We are referring to the value of index, not to items themselves, right?

(This would apply to all similar sentences)

Reply:
Yes, you are correct! I was wrong. Thank you for enlightening me!

Done in https://github.com/supercollider/supercollider/pull/6795/commits/2a9228e5383e35c71f971c05d2f8770dd69e9826!